### PR TITLE
Post-repo-split work

### PR DESCRIPTION
**Pull request by jplatte**

### CODE REVIEW

File: src/events.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //! Types related to events.
 
-use ruma::events::room::history_visibility::HistoryVisibility as RumaHistoryVisibility;
+use matrix_sdk_common::ruma::events::room::history_visibility::HistoryVisibility as RumaHistoryVisibility;
```
Comment (richvdh) [inline src/events.rs:3]: bit surprised about this. Why is using `matrix_sdk_common`'s re-export better than using ruma directly?

**Review by richvdh — APPROVED** [review]
lgtm

File: src/events.rs (hunk 1 above)
Comment (jplatte) [reply src/events.rs:3]: Don't have to keep versions in sync that way. Especially relevant when we're using a git dependency of Ruma in the SDK (which I hope to change soon-ish, but not exactly high priority as far as EX is concerned).

Comment (richvdh) [reply src/events.rs:3]: makes sense, thanks

**Pull request merged**